### DISCUSSION TITLE
MEETING: 24 January 2023 - EMu Hour

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#760] [MEETINGS]
# Agenda: EMu Hour
## Review creating Research Events
- We have had some requests for a refresher.
- https://github.com/fieldmuseum/EMu-Documentation/discussions/365

## Using Narratives for for verbatim label transcriptions - Moved to 31 January EMu-hour
- https://github.com/fieldmuseum/EMu-Documentation/discussions/753

## Dynamic Groups for IPT Reports
- Set up dynamic group/s for your IPT reports, and share with Admin. 
> (If you need help, see [“How to Save Search Criteria (Dynamic Group)” here](https://help.emu.axiell.com/v5.1/en/Topics/Common/How%20to%20manage%20groups%20in%20any%20module.htm)

## Poll for frequency of meetings 2023
[Google Poll](https://docs.google.com/forms/d/e/1FAIpQLSc7gaeI2SPcVLsWxGkMiycFt1GNj__2ZzEeEVEB_6IaeJ-wqw/viewform?usp=sf_link) - speak now or forever hold your peace

#### EMu basics
- [Signup](https://docs.google.com/spreadsheets/d/1ExCyeugFWHZypJhGo2aP42d4rPAaa4OYj0kKVheWh8o/edit#gid=391418581)
- REMINDER: if you don’t have an EMu account, please put in a helpdesk request to get one before the training session.
- - Action
- - Fossil Invertebrates
- - Exhibitions
#### none

**Comment by zkachian** [comment]
Action mammals sightings IPT group IRN = 15999
Action birds sightings IPT group IRN = 16000

**Reply by zkachian** [reply]
If something looks off with these groups, just let me know and I can update. Thanks.

**Comment by rondlg** [comment]
- [Question: 16 How to flag a record for future relationships? [RS] github #763](https://github.com/fieldmuseum/EMu-Documentation/discussions/763)
- [Question 17: Standards for audio recordings? [ZK - Action] githbu #764](https://github.com/fieldmuseum/EMu-Documentation/discussions/764)
